keps/sig-network/5495-deprecate-ipvs-mode-in-kube-proxy/README.md

```diff
@@ -0,0 +1,334 @@
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+This KEP proposes deprecation of `ipvs` in kube-proxy.
+
+## Motivation
+
+At time of writing, kube-proxy has four supported backends (`iptables`, `ipvs`, `nftables`, and `winkernel`).
+
+sig-network currently lacks maintainers who are familar with the `ipvs` backend code, and as such, has been encouraging
+users who report `ipvs` bugs to move to using the `nftables` backend mode, where they can. (ie: [first example], [second example])
+
+`ipvs` was introduced in [KEP-265] to solve performance problems in large clusters.
+[KEP-3866] was created to introduce a new `nftables` mode to kube-proxy. The goal of this new backend mode
+has always been to eventually replace `ipvs` and `iptables`[^1], as it solve the performance issues of iptables
+and already solves many of the bugs present in the `ipvs` mode.
+
+[First example]: https://github.com/kubernetes/kubernetes/issues/132689#issuecomment-3031585314
+[second example]: https://github.com/kubernetes/kubernetes/issues/132068#issuecomment-2945169346
+[^1]: See https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/3866-nftables-proxy/README.md#we-will-hopefully-be-able-to-trade-2-supported-backends-for-1
+
+[KEP-265]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/265-ipvs-based-load-balancing/README.md
+[KEP-3866]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/3866-nftables-proxy/README.md
+
+### Goals
+
+- Deprecate the `ipvs` mode of kube-proxy
+
+### Non-Goals
+
+- Removal of the `ipvs` mode of kube-proxy - The specifics of removal of `ipvs` are being handled in [KEP-5343]
+
+[KEP-5343]: https://github.com/kubernetes/enhancements/pull/5344
+
+## Proposal
+
+### Notes/Constraints/Caveats (Optional)
+
+The specifics of removal of `ipvs` mode will be handled in [KEP-5343]
+
+### Risks and Mitigations
+
+N/A
+
+## Design Details
+
+### Test Plan
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+N/A - This KEP is only adding a deprecation warning and updating necessary parts of the website
+
+##### Integration tests
+
+N/A - This KEP is only adding a deprecation warning and updating nessesary parts of the website
+
+##### e2e tests
+
+N/A - This KEP is only adding a deprecation warning and updating nessesary parts of the website
+
+### Graduation Criteria
+
+#### Deprecation
+
+- Update the kubernetes website with deprecation notices for `ipvs`
+- Print a warning when a user starts kube-proxy in `ipvs` mode
```

Comment:
past tense, since  you're talking about what has to have happened in order for the KEP to graduate. eg "The Kubernetes web site has been updated ..."